web/index.ts

```diff
@@ -69,9 +74,12 @@ createConnection().then(setupPDFGenerationEnvironment)
         addCorsMiddleware();
         addSecurityMiddleware();
 
-        configureParticipationCertificateAPI();
+
+        if (isCommandArg("--noPDF")) {
```

Comment:
Shouldn't this be `if (!isCommandArg(...))` ?

Reply:
Indeed, thanks! 😬